### PR TITLE
New Devices (Matter Switch) Zemismart Multiple Devices

### PR DESCRIPTION
**Pull request by lelandblue**
This is a WWST Cert Request, two of the three devices will be CxS. 

From the Matter DCL, here is additional information:

Device 1
Vendor ID | 5020 (0x139c)
Product ID | 53249 (0xd001)
Device Type ID | 269 (0x10d)

Device 2
Vendor ID | 5020 (0x139c)
Product ID | 53250 (0xd002)
Device Type ID | 269 (0x10d)

Device 3
Vendor ID | 5020 (0x139c)
Product ID | 53251 (0xd003)
Device Type ID | 269 (0x10d)

### CODE REVIEW

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 1]
```diff
@@ -2080,6 +2080,22 @@ matterManufacturer:
     vendorId: 0x139C
     productId: 0xEEE2
     deviceProfileName: switch-binary
+  - id: "5020/53249"
+    deviceLabel: Zemismart downlight
+    vendorId: 0x139C
+    productId: 0xD001
+    deviceProfileName: light-color-level
+  - id: "5020/53250"
+    deviceLabel: Zemismart bulb
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/fingerprints.yml:2089]:
```suggestion
    deviceLabel: Zemismart Bulb
```
Should this be capitalized? That seems to be the format followed throughout

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 2]
```diff
@@ -2080,6 +2080,22 @@ matterManufacturer:
     vendorId: 0x139C
     productId: 0xEEE2
     deviceProfileName: switch-binary
+  - id: "5020/53249"
+    deviceLabel: Zemismart downlight
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/fingerprints.yml:2084]:
```suggestion
    deviceLabel: Zemismart Downlight
```
Same as below

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 3]
```diff
@@ -2080,6 +2080,22 @@ matterManufacturer:
...
+    vendorId: 0x139C
+    productId: 0xD001
+    deviceProfileName: light-color-level
+  - id: "5020/53250"
+    deviceLabel: Zemismart bulb
+    vendorId: 0x139C
+    productId: 0xD002
+    deviceProfileName: light-color-level
+  - id: "5020/53251"
+    deviceLabel: Zemismart Led Light Strip Driver
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/fingerprints.yml:2094]:
```suggestion
    deviceLabel: Zemismart LED Light Strip Driver
```
Should this be LED?

Comment (lelandblue) [reply drivers/SmartThings/matter-switch/fingerprints.yml:2094]: This is what the Partner has provided, but I think we can make that improvement.

**Review by ctowns — APPROVED** [review]

**Pull request merged**